### ISSUE TITLE
Support changing texture filtering

### ISSUE THREAD

**Issue by ooploopl-** [patriciogonzalezvivo/glslViewer#152]
As far as i understand there is no support for changing filtering when loading textures.

**Comment by patriciogonzalezvivo** [comment]
No... but you can cheat it some extra code. For example to use nearest you need to decimate the UV: 
`
uv = floor(uv*scale)/scale;
`

**Comment by ooploopl-** [comment]
I think it is nice to have filtering, wrapping options for textures so you can get identical results between this tool, your app vice versa. I already convert it to pixels by code provided, just wanted to suggest using hardware features. This may open the path for more features later on.
